server/server.js

```diff
@@ -15,4 +15,18 @@ app.get('/api/sites/:siteId', (req, res) => {
   })
 })
 
+app.get('/api/sites/:siteId/recommend', (req, res) => {
+  console.log(`receiving a get request for recommendations for site: ${req.params.siteId}`);
+
+  db.siteRecommend(req.params.siteId, (err, result) => {
+    if (err) throw err;
+    console.log('sending the results');
+    res.send(result);
+  })
+})
+
+app.get('*', (req, res) => {
```

Comment:
What is this get request for?

Reply:
@claire-melbourne This is the catchall get request! Without this, If I went to localhost:3003/5, I would get an error that the browser could not return a get request to that link. With this, any params are served with the index.html.